install.ps1

```diff
@@ -26,22 +26,20 @@ function downloadDvm([string] $dvmDir) {
   if( [System.Environment]::Is64BitOperatingSystem ) { $arch = "amd64" } else { $arch = "386"}
 
   # Download latest release
-  $latestTag = (ConvertFrom-Json (wget https://api.github.com/repos/getcarina/dvm/tags).Content) | select -ExpandProperty name | select -first 1
-
-  $webClient.DownloadFile("https://github.com/getcarina/dvm/releases/download/$latestTag/dvm-helper-windows-$arch.exe", "$tmpDir\dvm-helper-windows-$arch.exe")
-  $webClient.DownloadFile("https://github.com/getcarina/dvm/releases/download/$latestTag/dvm-helper-windows-$arch.exe.sha256", "$tmpDir\dvm-helper-windows-$arch.exe.256")
+  $webClient.DownloadFile("https://download.getcarina.com/dvm/latest/windows/$arch/dvm-helper.exe", "$tmpDir\dvm-helper.exe")
+  $webClient.DownloadFile("https://download.getcarina.com/dvm/latest/windows/$arch/dvm-helper.exe.sha256", "$tmpDir\dvm-helper.exe.256")
```

Comment:
These need to be using the `x86_64` version for the paths rather than the golang arch.